sympy/vector/functions.py

```diff
@@ -214,6 +214,38 @@ def gradient(scalar, coord_sys):
 
     return coord_sys.delop(scalar).doit()
 
+def directional_derivative(scalar, vect, coord_sys):
+    """
+    Returns the directional derivative of a scalar field computed along a  given vector
```

Comment:
Here you have double white space, after "along a".
If you want to correct this, you shouldn't push new commit. You can use `git commit --amend`. It overrides before one.